mkdocs.yml

```diff
@@ -78,11 +80,12 @@ not_in_nav: |
 
 # Page tree
 nav:
-  - Home: index.md
+  - Home: Home/index.md
```

Comment:
I don't understand why you made this change

Reply:
In my last PR I introduced snippets (parts of code to use in different places) for the list of pipelines and converters in Clinica. This way, it only needs to be updated in one place and can be re-used as needed anywhere in the documentation. The thing is, it contains relative links to the different pages (ex : [ADNI-to-BIDS] ( ../Converters/ADNI2BIDS.md ) ) so if I want to use the same snippet I need to make sure the different files where it is used is placed at the same relative "level" compared to the links it points to.

These snippets with links were used in the .md files from the docs/Software repository and the docs/index.md. The easiest way I found to keep these (because it seems easy to maintain) is to place the index.md in another folder.

Reply:
Alright ! Thanks for the explanations 👍